File: InschrijvingenApp/Entities/Inschrijving.cs
using DataAccess.Entities;

namespace InschrijvingPietieterken.Entities
{
    public class Inschrijving : EntityBase
    {
        public Inschrijving()
        {
            Kind = new Kind();
            Ouders = new Ouders();
            Medisch = new Medisch();
        }

        public int KindId { get; set; }
        public Kind Kind { get; set; }

        public int OudersId { get; set; }
        public Ouders Ouders { get; set; }

        public int MedischId { get; set; }
        public Medisch Medisch { get; set; }

        public string OverigeInfo { get; set; }
    }
}

File: InschrijvingenApp/ClientApp/app/components/models/inschrijving.ts
import { Kind } from "./kind";
import { Ouders } from "./ouders";
import { Medisch } from "./medisch";

export class Inschrijving {

    constructor() {
        this.kind = new Kind();
        this.ouders = new Ouders();
        this.medisch = new Medisch();
    }

    public kind: Kind;
    public ouders: Ouders;
    public medisch: Medisch;
    public overigeInfo: string = "";
    public heeftMaandAbonnement: boolean = false;
}